cc-backend/openapi.yaml

```diff
@@ -0,0 +1,406 @@
+              schema:
+                type: string
+
+  /v1/user:
+    post:
+      operationId: afterLogin
+      summary: User has logged in
+      description: TODO
+      tags:
+        - User
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserInfo"
+        description: Payload
+        required: true
+      responses:
+        "200":
+          description: Added User info
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserInfo"
+        "400":
+          description: Invalid request was sent to the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "403":
+          description: Attempted to make a call with unauthorized client
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "404":
+          description: Not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /v1/items/{surveyId}:
+    get:
+      operationId: getItemsFromPK
+      summary: Get all items from PK
+      description: Gets all items from PK via CC-PK-service
+      tags:
+        - Item
+      parameters:
+        - name: surveyId
+          required: true
+          in: path
+          description: surveyId from which to list reusable items
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Items
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Items"
+        "400":
+          description: Invalid request was sent to the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "403":
+          description: Attempted to make a call with unauthorized client
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "404":
+          description: Not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  
+  /v1/items/{itemId}:
+    get:
+      operationId: getItemInfoForm
+      summary: Gets item info for form from PK
+      description: Gets item info for Form from PK via CC-PK-service
```

Comment:
description: also get possible Item configurations and drafts from CC-db-service